app/forms/mappings/LocalDateFormatter.scala

```diff
@@ -108,6 +107,52 @@ private[mappings] class LocalDateFormatter(
       s"$key.year" -> value.getYear.toString
     )
 
+    /**
+   * Updates the FormError key as per valid day, month and year criteria
+   * key is updated as below
+   * invalid day - key.day
+   * invalid month - key.month
+   * invalid year - key.year
+   *
+   * @param key Input form key
+   * @param day Day value of the input date
+   * @param month Month value of the input date
+   * @param year Year value of the input date
+   * @return Updated form key
+   */
+  private[mappings] def updateFormErrorKeys(key: String,
+                                            month: Int,
+                                            year: Int): String =
+    (month, year) match {
+      case (m, _) if m < 1 || m > 12 => s"$key.month"
+      case (_, y) if y < 1000 || y > 99999 => s"$key.year"
+      case _ => s"$key.day"
+    }
+
+  /**
+   * Updates the FormError key as per the given criteria
+   * key is updated as below
+   * empty day or non numeric day - key.day
```

Comment:
Could you please update the comments as we don't have day here? So day reference must be removed